packages/combo-box/src/vaadin-combo-box.js

```diff
@@ -121,6 +121,9 @@ registerStyles('vaadin-combo-box', inputFieldShared, { moduleId: 'vaadin-combo-b
  * `opened`  | Set when the combo box dropdown is open | :host
  * `loading` | Set when new items are expected | :host
  *
+ * If you want to replace the default `<input>` and its container with a custom implementation to get full control
+ * over the input field, consider using the [`<vaadin-combo-box-light>`](#/elements/vaadin-combo-box-light) element.
+ *
```

Comment:
Not sure how this got in here, but can be removed.

Reply:
Done.